src/Terminal/HelpText.ts

```diff
@@ -295,28 +295,30 @@ export const HelpTexts: IMap<string[]> = {
     " ",
   ],
   ls: [
-    "Usage: ls [dir] [| grep pattern]",
+    "Usage: ls [dir] [--grep pattern] | [-l]",
     " ",
     "The ls command, with no arguments, prints all files and directories on the current server's directory to the Terminal screen. ",
     "The files will be displayed in alphabetical order. ",
     " ",
     "The 'dir' optional parameter can be used to display files/directories in another directory.",
     " ",
-    "The '| grep pattern' optional parameter can be used to only display files whose filenames match the specified pattern.",
+    "The '--grep pattern' optional parameter can be used to only display files whose filenames match the specified pattern.",
+    " ",
+    "The '-l' optional parameter allows you to force each item onto a single line.",
```

Comment:
Swap lines 305 and 307 to stay in line with the other places where `-l` comes first